test/e2e/tests/privacy-mode/privacy-mode.spec.js

```diff
@@ -1,106 +0,0 @@
-const { strict: assert } = require('assert');
-const {
-  withFixtures,
-  unlockWallet,
-  defaultGanacheOptions,
-} = require('../../helpers');
-const FixtureBuilder = require('../../fixture-builder');
-
-describe('Privacy Mode', function () {
-  it('should activate privacy mode, then deactivate it', async function () {
-    await withFixtures(
-      {
-        dapp: true,
-        fixtures: new FixtureBuilder().withPreferencesController().build(),
-        ganacheOptions: defaultGanacheOptions,
-        title: this.test.fullTitle(),
-      },
-      async ({ driver }) => {
-        async function checkForHeaderValue(value) {
-          const balanceElement = await driver.findElement(
-            '[data-testid="account-value-and-suffix"]',
-          );
-          const surveyText = await balanceElement.getText();
-          assert.equal(
-            surveyText,
-            value,
-            `Header balance should be "${value}"`,
-          );
-        }
-
-        async function checkForTokenValue(value) {
-          const balanceElement = await driver.findElement(
-            '[data-testid="multichain-token-list-item-value"]',
-          );
-          const surveyText = await balanceElement.getText();
-          assert.equal(surveyText, value, `Token balance should be "${value}"`);
-        }
-
-        async function checkForPrivacy() {
-          await checkForHeaderValue('••••••');
-          await checkForTokenValue('••••••');
-        }
-
-        async function checkForNoPrivacy() {
-          await checkForHeaderValue('25');
-          await checkForTokenValue('25 ETH');
-        }
-
-        async function togglePrivacy() {
-          const balanceElement = await driver.findElement(
-            '[data-testid="account-value-and-suffix"]',
-          );
-          const initialText = await balanceElement.getText();
-
-          await driver.clickElement('[data-testid="sensitive-toggle"]');
-          await driver.wait(async () => {
-            const currentText = await balanceElement.getText();
-            return currentText !== initialText;
-          }, 2e3);
-        }
-
-        await unlockWallet(driver);
-        await checkForNoPrivacy();
-        await togglePrivacy();
-        await checkForPrivacy();
-        await togglePrivacy();
-        await checkForNoPrivacy();
-      },
-    );
-  });
-
-  it('should hide fiat balance and token balance when privacy mode is activated', async function () {
```

Comment:
the first spec and the second spec where identical, with the difference that the second spec opened the account menu and checked the balances there too.
The 2 spec have been combined now and a new spec has been added to split up the coverage like:
- first spec, from privacy off (by default, so no custom fixtures needed), we activate the privacy mode, and check that all balance ocurrences, including the account menu, are hidden
- second spec, from privacy on (uses fixtures), we deactivate the privacy mode, and check that all balance occurrences are visible